.storybook/preview.js

```diff
@@ -27,3 +28,20 @@ for (let i = -3; i <= 6; i++) {
 }
 addDecorator(withPaddings);
 addParameters({ paddings });
+
+// Create viewports using widths defined in design tokens
+const breakpointViewports = Object.keys(breakpoints).map((name) => {
+  return {
+    name,
```

Comment:
Could we set the name to be something like:

```js
{
  name: `breakpoints.${name}`
}
```

This would align with our usage of breakpoints in Sass, which might be more helpful?

Reply:
That sounds good. Thanks for the suggestion.